proposals/memory-specializations/utilities.tex

```diff
@@ -0,0 +1,349 @@
+%!TEX root = std.tex
+\setcounter{chapter}{19}
+\rSec0[utilities]{General utilities library}
+
+\setcounter{section}{6}
+\rSec1[memory]{Memory}
+\synopsis{Header \tcode{<\added{experimental/ranges/}memory>} synopsis}
+
+\setcounter{subsection}{11}
+\rSec2[specialized.algorithms]{Specialized algorithms}
+
+\rSec3[uninitialized.construct.default]{uninitialized_default_construct}
+\begin{codeblock}
+namespace std { @\added{namespace experimental \{ namespace ranges \{ inline namespace v1 \{}@
+  template <@\changed{class ForwardIterator}{OutputIterator O, Sentinel<O> S}@>
+    @\added{requires DefaultConstructible<value_type_t<O>{>}() \&\&}@
+             @\added{Same<value_type_t<O>\&, reference_t<O>{>}}@
```

Comment:
> You're missing parens for Same</*...*/>()
Oops.

> This requirement comes up repeatedly throughout this section. We should consider defining an exposition-only concept

Putting this in, but why can't this be a (separate) proposal for `<ranges/concepts>`? It seems like it could be useful elsewhere.